code/modules/reagents/chemistry/reagents/drinks.dm

```diff
@@ -317,12 +317,12 @@
 	name = "Soy Latte"
 	id = "soy_latte"
 	description = "A nice and tasty beverage while you are reading your hippie books."
-	color = "#664300" // rgb: 102, 67, 0
+	color = "#8A6723"
 	adj_sleepy = 0
 	drink_icon = "soy_latte"
 	drink_name = "Soy Latte"
 	drink_desc = "A nice and refrshing beverage while you are reading."
-	taste_description = "fake milky coffee"
+	taste_description = "milkish coffee"
```

Comment:
```suggestion
	taste_description = "milky coffee"
```
milky would be the proper grammar here

Reply:
It would, though I deliberately wanted to evoke the "fake" part of it by using the incorrect wording!